packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/upgrade.html

```diff
@@ -0,0 +1,138 @@
+<oui-header
+    data-heading="{{:: ('dedicated_server_dashboard_upgrade_title_' + $ctrl.upgradeType) | translate }}"
+    description="{{:: !$ctrl.upgradeStarted ? (('dedicated_server_dashboard_upgrade_description_' + $ctrl.upgradeType) | translate) : '' }}"
+>
+    <oui-back-button
+        data-on-click="$ctrl.goBack()"
+        previous-page="{{:: 'dedicated_server_dashboard_upgrade_back_text' | translate }}"
+    >
+    </oui-back-button>
+</oui-header>
+
+<oui-message data-type="info">
+    <span data-translate="dedicated_server_dashboard_upgrade_info"></span>
+</oui-message>
+
+<oui-stepper data-current-index="$ctrl.currentStep">
+    <oui-step-form
+        data-header="{{:: ('dedicated_server_dashboard_upgrade_select_option_title_' + $ctrl.upgradeType) | translate }}"
+        data-description="{{:: ('dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType) | translate }}"
+        data-submit-text="{{:: 'dedicated_server_dashboard_upgrade_order' | translate }}"
+        data-on-submit="$ctrl.orderOption()"
+        data-disabled="$ctrl.upgradeStarted"
+        data-valid="$ctrl.selectedOption || $ctrl.upgradeStarted"
+        data-editable="false"
+        data-prevent-next
+    >
+        <p
+            data-translate="dedicated_server_dashboard_upgrade_select_option_description"
+        ></p>
+        <oui-datagrid data-rows="$ctrl.options">
+            <oui-datagrid-column
+                data-title=":: 'dedicated_server_dashboard_upgrade_select_option_name' | translate"
+            >
+                <oui-radio
+                    data-name="selectedOption"
+                    data-model="$ctrl.selectedOption"
+                    data-value="$row"
+                    ><span data-ng-bind="$row.productName"></span>
+                </oui-radio>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'dedicated_server_dashboard_upgrade_select_option_price' | translate"
+            >
+                <span data-ng-bind="$ctrl.getOptionPrice($row).text"></span>
+            </oui-datagrid-column>
+        </oui-datagrid>
+        <oui-spinner
+            data-ng-if="$ctrl.orderInProgress"
+            data-size="m"
+        ></oui-spinner>
+    </oui-step-form>
+    <oui-step-form
+        data-ng-if="$ctrl.upgradeStarted"
+        data-disabled="$ctrl.upgradeScheduled"
+        data-header="{{:: 'dedicated_server_dashboard_upgrade_intervention_title' | translate }}"
+        data-on-focus="$ctrl.prepareInterventionData()"
+        data-submit-text="{{:: 'common_confirm' | translate }}"
```

Comment:
**question:** Can this module bring its own translations?

Reply:
This is handled. Thanks